application/src/main/java/org/opentripplanner/graph_builder/module/osm/OsmDatabase.java

```diff
@@ -787,9 +787,11 @@ private void processMultipolygonRelations() {
    * Handler for a new Area (single way area or multipolygon relations)
    */
   private void newArea(Area area) {
```

Comment:
You could rename this method as this now currently sounds like a getter method without a verb.